src/browser/of_events.ts

```diff
@@ -42,6 +42,15 @@ class OFEvents extends EventEmitter {
 
         return super.emit(route, ...data);
     }
+
+    public SUBSCRIBER: StringMap = {
```

Comment:
Regarding UPPERCASE, yea it's constants. Too old school?
```js
ofEvents.SUBSCRIBER.ADDED
ofEvents.SUBSCRIBER.added
ofEvents.subscriber.ADDED
ofEvents.subscriber.added
```
I like 1; I could live with 3; I don't like 2 and 4.